spinal_cord/sub-amu01_T2w_PAM50.csv

```diff
@@ -1,885 +1,946 @@
-"2023-02-22 12:19:11","git-master-aef83eb1482ea9360065ac8d8913aaf18edd4ce8","sub-amu01/anat/sub-amu01_T2w_seg.nii.gz","955","1","",81.14770661389241,0.0,-2.173895871804289,0.0,-2.4403054453943867,0.0,8.63920620967338,0.0,11.914417608729895,0.0,0.6886151599866748,0.0,3.185481408922489,0.0,0.9594402758395628,0.0,""
-"2023-02-22 12:19:11","git-master-aef83eb1482ea9360065ac8d8913aaf18edd4ce8","sub-amu01/anat/sub-amu01_T2w_seg.nii.gz","956","1","",81.15720328089168,0.0,-2.2127879677664706,0.0,-2.236062578354883,0.0,8.683271140219228,0.0,11.870839526152666,0.0,0.6818037618499017,0.0,3.8432597538490083,0.0,0.9632048647200162,0.0,""
-"2023-02-22 12:19:11","git-master-aef83eb1482ea9360065ac8d8913aaf18edd4ce8","sub-amu01/anat/sub-amu01_T2w_seg.nii.gz","957","1","",81.46688819950664,0.0,-2.25178769069111,0.0,-2.0276303377952263,0.0,8.708229067780326,0.0,11.89183250204047,0.0,0.6809146352751,0.0,3.745835192436285,0.0,0.9652656124386939,0.0,""
-"2023-02-22 12:19:11","git-master-aef83eb1482ea9360065ac8d8913aaf18edd4ce8","sub-amu01/anat/sub-amu01_T2w_seg.nii.gz","958","1","",82.2018398079105,0.0,-2.2909398851458973,0.0,-1.8132631514153528,0.0,8.706118741113034,0.0,12.004301144087073,0.0,0.688415393413091,0.0,2.578539847042982,0.0,0.9649125851781893,0.0,""
-"2023-02-22 12:19:11","git-master-aef83eb1482ea9360065ac8d8913aaf18edd4ce8","sub-amu01/anat/sub-amu01_T2w_seg.nii.gz","959","1","",82.69761440952642,0.0,-2.3302623658264934,0.0,-1.592266220171192,0.0,8.719965023744097,0.0,12.062758887724355,0.0,0.6909714593153237,0.0,1.6873958256116712,0.0,0.9652226244600937,0.0,""
-"2023-02-22 12:19:11","git-master-aef83eb1482ea9360065ac8d8913aaf18edd4ce8","sub-amu01/anat/sub-amu01_T2w_seg.nii.gz","960","1","",83.0675063759908,0.0,-2.369674470836462,0.0,-1.3677799495247758,0.0,8.742209521795345,0.0,12.092789842725152,0.0,0.6909250556197888,0.0,0.9415946062656231,0.0,0.9658816461327395,0.0,""
-"2023-02-22 12:19:11","git-master-aef83eb1482ea9360065ac8d8913aaf18edd4ce8","sub-amu01/anat/sub-amu01_T2w_seg.nii.gz","961","1","",82.45779158803484,0.0,-2.4093194260998767,0.0,-1.134230514353658,0.0,8.691472398129235,0.0,12.08486860880659,0.0,0.6947845911808981,0.0,1.0821814833541663,0.0,0.9590614058782556,0.0,""
-"2023-02-22 12:19:11","git-master-aef83eb1482ea9360065ac8d8913aaf18edd4ce8","sub-amu01/anat/sub-amu01_T2w_seg.nii.gz","962","1","",81.6845588368763,0.0,-2.449027044497151,0.0,-0.8982431689399251,0.0,8.624468555255705,0.0,12.076005050408304,0.0,0.6999242640343872,0.0,1.2985618065375693,0.0,0.9521664432719886,0.0,""
-"2023-02-22 12:19:11","git-master-aef83eb1482ea9360065ac8d8913aaf18edd4ce8","sub-amu01/anat/sub-amu01_T2w_seg.nii.gz","963","1","",80.59578599437025,0.0,-2.4889583866251943,0.0,-0.6535548204651668,0.0,8.508428694535834,0.0,12.088621331020914,0.0,0.7102480118474784,0.0,1.3494289828850732,0.0,0.9501981644722192,0.0,""
+2025-01-30 15:37:23,git-master-310834f50bfc9e3f9c68d696ada10a2223a882de*,sub-amu01/anat/sub-amu01_T2w_label-SC_seg.nii.gz,40,20,,,,,,,,,,,,,,,,,,
```

Comment:
I'm now thinking about one thing: the CSV files contain the SCT version, e.g., `git-master-310834f50bfc9e3f9c68d696ada10a2223a882de*`, but not the information about the exact model. In other words, even with the latest SCT version, we could have used `sct_deepseg_sc`. I know that the model we used is tracked by the JSON sidecars, but we do not have them available here, so we're kind of losing this information.

Reply:
hmm, I am wondering what we can do about it, since this part is only associated with `sct_process_segmentation`, we can discuss of ideas at our next meeting maybe

Reply:
Maybe we could simply add a new column with the model name?

Reply:
okay, resolving this conversation as we agreed to track the model and its version using releases. context: https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4810#issuecomment-2712045286